vita3k/config/include/config/config.h

```diff
@@ -57,5 +57,5 @@ struct Config {
     bool color_surface_debug = false;
     bool hardware_flip = false;
     bool performance_overlay = false;
-
+    std::string backend = {};
```

Comment:
change here
```suggestion
    std::string backend_render = "OpenGL";
```

Reply:
I'm not doing that. It'll screw up the command line and you wouldn't be able to switch to Vulkan vis config.yml. This needs (should) to be empty by default!

Reply:
no, sorry, no set empty, no have any sense, is obligatory one things writed, is OpenGL or Vulkan, but never empty, and no broken command line is same all other

Reply:
We set the default [here](https://github.com/Vita3K/Vita3K/pull/568/files#diff-a6a8977717cb5a4e9908c4f9b0256552R135). app_init.cpp makes sure that you select either. I really don't think we need this.

Reply:
yes, i know, just is more sense and more good writed, default value is also need for create file no forget, for not set like i have myself forget set before and get icon size on 0 on create config lol set value is needed

Reply:
i have all time right haha, exactly like i have explain, is fail config file, keep backend entry empty, write "OpenGL" here
![image](https://user-images.githubusercontent.com/5261759/64374277-c5a70180-d023-11e9-8fa9-3eb74429f0be.png)